Session03/Session03/Session03Excercise02/Program.cs

```diff
@@ -7,14 +7,44 @@ class Program
         static void Main(string[] args)
         {
             Console.WriteLine("Ange ett antal siffror, separerat med kommatecken.");
-            var input = Console.ReadLine();
-            var inputArray = input.Split(new [] { ',' }, StringSplitOptions.RemoveEmptyEntries);
-
-            foreach (var number in inputArray)
+            string input = Console.ReadLine();
+            string[] inputArray = input.Split(".", StringSplitOptions.RemoveEmptyEntries);
+            double sum = 0;
+            double highestValue = 0;
+            double lowestValue = 0;
+            double?[] numberArray = new double?[inputArray.Length];
+            for (int i = 0; i < inputArray.Length; i++)
             {
-                Console.WriteLine("Värdet är " + number);
+                bool parsed = double.TryParse(inputArray[i], out double parsedValue);
+
+                if (parsed)
+                { 
+                    Console.WriteLine(parsedValue);
+                    highestValue = parsedValue;
+                    lowestValue = parsedValue;
+                    numberArray[i] = parsedValue;
+                }
+                else numberArray[i] = null;
+
             }
 
+            foreach (var number in numberArray)
+            {
+                if (number != null)
+                {
+                    sum += (double)number;
+                    foreach (var number2 in numberArray)
+                    {
+                        if ((number > number2) & (number > highestValue)) highestValue = (double)number;
```

Comment:
För logiska jämförelser använder man `&&`.
```(number > number2) && (number > highestValue)```

Ovanstående kod fungerar, men det som händer bakom kulisserna är att det binära värdet slås ihop via bitwise-operation.
Här är en sida som beskriver skillnaden:
https://stackoverflow.com/questions/5051349/what-is-the-difference-between-bitwise-and-and-logical-and